doc/compiler/CompilerOptions.md

```diff
@@ -135,7 +135,7 @@ options, etc).
 | disableGRA                                       | disable IL based global register allocator                                      |
 | disableInlining                                  | disable IL inlining                                                             |
 | disableLiveRegisterAnalysis                      | disable live register analysis                                                  |
-| disableLoopUnroller                              | disable loop unroller                                                           |
+| disableGLU                                       | disable general loop unroller                                                   |
```

Comment:
This table is in alphabetical order.  Can you add the corrected name in the appropriate spot please?